test/nursery/test_autogluon_tabular.py

```diff
@@ -173,23 +175,31 @@ def test_get_features_dataframe(
 )
 @pytest.mark.parametrize("lag_indices", [[], [1, 2, 5]])
 @pytest.mark.parametrize("disable_auto_regression", [False, True])
+@pytest.mark.parametrize("path", [Path("test_autogluon_dir")])
+@pytest.mark.parametrize("ag_path_prefix", [None, Path("test_autogluon_dir")])
 def test_tabular_estimator(
     dataset,
     freq,
+    path,
+    ag_path_prefix,
     prediction_length: int,
     lag_indices: List[int],
     disable_auto_regression: bool,
 ):
     estimator = TabularEstimator(
         freq=freq,
         prediction_length=prediction_length,
+        ag_path_prefix=ag_path_prefix,
         lag_indices=lag_indices,
-        time_limits=10,
+        time_limit=10,
         disable_auto_regression=disable_auto_regression,
     )
 
+    os.makedirs(path, exist_ok=True)
```

Comment:
If you go the tempdir way (see comment above) there should be no way to create any directory explicitly

Reply:
I'm confused about what this means..